app/views/capture-debt.njk

```diff
@@ -0,0 +1,145 @@
+{% extends '_layout.njk' %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "./macros/radio-list.njk" import radioList %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+      text: "Back",
+      href: "/",
+      attributes: {
+        id: "back"
+      }
+    }) 
+  }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Create new reporting dataset</h1>
+    </div>
+    <div class="govuk-grid-column-two-thirds">
+
+    {% if model.errorMessage %}
+      {{ govukErrorSummary({
+        titleText: "Something went wrong",
+        errorList: [
```

Comment:
Could we load the errors into an array instead of a single text string? At the moment the error message is displayed in one big text block:

"The FRN cannot be empty. The application identifier cannot be empty. The net value cannot be empty. The type of debt is invalid. The debt day cannot be empty. The debt month cannot be empty. The debt year cannot be empty"

When there are errors and I fill in one of the fields and click "Continue", the value I have just entered doesn't persist. So you need to send back the value and populate the form component:

```
{{ govukInput({
            label: {
              text: "Firm Reference Number (FRN)",
              classes: "govuk-label--s"
            },
            classes: "govuk-input--width-10",
            id: "frn",
            name: "frn"
            value: model.frn
          }) 
        }}
```

As this is internal, if I use an incorrect format, that the error message gives me clearer information of what the value is meant to be.

eg. 

The FRN is too short. This is a 10 digit number
The application identifier is too short. . This is a 10 digit number

Is there validation for how far in the past a debt can be discovered?